Cabal/src/Distribution/PackageDescription/Check/Monad.hs

```diff
@@ -37,6 +37,7 @@ module Distribution.PackageDescription.Check.Monad
   , checkP
   , checkPkg
   , liftInt
+  , liftCM
```

Comment:
If we don't want to expose this I can probably replace with a function *like* `liftInt` but that allows a secondary return, like,

```haskell
liftIntWith 
    :: (CheckInterface m -> Maybe (i m))
    -> (i m -> m ([PackageCheck], r)
    -> CheckM m (Maybe r)
```

Reply:
Exposing it is fine.